vividus-util/src/main/java/org/vividus/util/qrCode/QRCodeReader.java

```diff
@@ -0,0 +1,34 @@
+package org.vividus.util.qrCode;
+
+import com.google.zxing.BinaryBitmap;
+import com.google.zxing.LuminanceSource;
+import com.google.zxing.MultiFormatReader;
+import com.google.zxing.NotFoundException;
+import com.google.zxing.Result;
+import com.google.zxing.client.j2se.BufferedImageLuminanceSource;
+import com.google.zxing.common.HybridBinarizer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+import java.nio.file.Path;
+
+public class QRCodeReader {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(QRCodeReader.class);
+
+    public String readQRCode(Path path) throws IOException {
+        try {
+            BufferedImage bufferedImage = ImageIO.read(path.toFile());
+            LuminanceSource source = new BufferedImageLuminanceSource(bufferedImage);
+            BinaryBitmap bitmap = new BinaryBitmap(new HybridBinarizer(source));
+            Result result = new MultiFormatReader().decode(bitmap);
+            return result.getText();
+        } catch (NotFoundException e) {
+            LOGGER.error("There is no QR code in the image");
+            return "";
```

Comment:
the error should be thrown explicitly, we should avoid error conventions for `return`-s